apps/web/src/pages/quick-start/steps/Trigger.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { QuickStartWrapper } from '../components/QuickStartWrapper';
+import { CheckCircleBroken } from '../../../design-system/icons/gradient/CheckCircleBroken';
+import { TestNotificationTrigger } from '../components/TestNotificationTrigger';
+import { BellGradient } from '../../../design-system/icons';
+
+export function Trigger() {
+  return (
+    <QuickStartWrapper
+      title={<CheckCircleBroken style={{ height: '75px', width: '75px', marginTop: '100px' }} />}
+      secondaryTitle={'Amazing, nearly done!'}
+      description={<TriggerDescription />}
+      faq={true}
+    >
+      <TestNotificationTrigger />
+    </QuickStartWrapper>
+  );
+}
+
+export function TriggerDescription() {
+  return (
+    <span>
+      Now let's ring the nc in your app
+      <BellGradient style={{ margin: '0px 5px 0', top: '8px', position: 'relative' }} />
+      in your app
```

Comment:
```suggestion
      Now let's ring the bell in your new notification center
      <BellGradient style={{ margin: '0px 5px 0', top: '8px', position: 'relative' }} />
```

Reply:
Just to make sure which one did you mean the one with the icon in the middle or at the end?
@scopsy 

![image](https://user-images.githubusercontent.com/39195835/219175498-6cf12b42-6069-43a2-8e9a-d16718526157.png)


![image](https://user-images.githubusercontent.com/39195835/219175641-b2d68c56-5625-482b-821c-3e4c3c4bd015.png)

Reply:
In the end 🙏

Reply:
np